code/_globalvars/traits.dm

```diff
@@ -186,7 +186,9 @@ GLOBAL_LIST_INIT(traits_by_type, list(
 		"TRAIT_ARTIST" = TRAIT_ARTIST,
 		"TRAIT_GUNS" = TRAIT_GUNS,               // used for the "no guns" challenge
 		"TRAIT_HEALING" = TRAIT_HEALING,          // used for the "no medipens" challenge
-		"TRAIT_GRAB_IMMUNE" = TRAIT_GRAB_IMMUNE //used to prevent grab-related exploits
+		"TRAIT_GRAB_IMMUNE" = TRAIT_GRAB_IMMUNE, //used to prevent grab-related exploits
+		"TRAIT_PHYSICAL_HEALING_BLOCKED" = TRAIT_PHYSICAL_HEALING_BLOCKED,
```

Comment:
I was thinking about this but i kept forgetting to talk about it this will be very intresting to use.